tensorflow-2.2.0.patch

```diff
@@ -0,0 +1,12 @@
+diff --color --unified --recursive --text src/tensorflow-2.2.0/tensorflow/stream_executor/gpu/gpu_types.h src/tensorflow-2.2.0.new/tensorflow/stream_executor/gpu/gpu_types.h
+--- src/tensorflow-2.2.0/tensorflow/stream_executor/gpu/gpu_types.h	2020-05-05 22:58:49.000000000 +0100
++++ src/tensorflow-2.2.0.new/tensorflow/stream_executor/gpu/gpu_types.h	2020-07-14 21:07:05.406803883 +0100
+@@ -24,7 +24,7 @@
+ 
+ #include "rocm/include/hip/hip_complex.h"
+ #include "rocm/include/hip/hip_runtime.h"
+-#include "rocm/include/hiprand/hiprand.h"
++#include "rocm/hiprand/include/hiprand.h"
```

Comment:
Instead of patching the tensorflow source code it is better for us to add a symlink on the rocm-arch side. See this commit as reference: https://github.com/rocm-arch/rocm-arch/commit/b120d233c87dfd99362fa450e621367e5dbb4139
Can you make a PR to rocrand that resolves this issue so we do not need to patch tensorflow code?

Reply:
Ok, I'll try that. Should we keep that change until it is solved on their side?

Reply:
I was hoping that you would change it. Since I maintain both this repo and the rocm-arch repo I consider both projects on our side. It should be simple to implement based on the reference commit I showed you. I do not want this patch in when we merge your PR, since the problem is not here but rather in the rocrand Arch package, so we fix it there.

Reply:
I believe I made a PR to fix this properly, apologies if not

Reply:
With https://github.com/rocm-arch/rocm-arch/pull/300 this patch should not be needed now. If you could remove it that would be great.